curation_concerns-models/app/jobs/ingest_local_file_job.rb

```diff
@@ -22,16 +22,18 @@ def run
 
     if actor.create_content(File.open(path), filename, mime_type(filename))
       FileUtils.rm(path)
-      CurationConcerns.queue.push(ContentDepositEventJob.new(generic_file.id, user_key))
 
-      message = "The file (#{File.basename(filename)}) was successfully deposited."
-      subject = 'Local file ingest'
```

Comment:
This is to be added as a method called by an event in sufia_events.rb.
See:
https://github.com/projecthydra-labs/sufia-core/blob/master/config/initializers/sufia_events.rb
projecthydra-labs/sufia-core/issues/6

The idea was that user messaging via email doesn't belong in curation_concerns.